cmd/feed-ingress/main.go

```diff
@@ -219,6 +219,9 @@ func init() {
 		"Path to OpenTracing plugin on disk (eg. /usr/local/lib/libjaegertracing_plugin.so)")
 	flag.StringVar(&nginxOpenTracingConfigPath, "nginx-opentracing-config-path", defaultNginxOpenTracingConfigPath,
 		"Path to OpenTracing config on disk (eg. /etc/jaeger-nginx-config.json)")
+	flag.IntVar(&nginxConfig.ClientHeaderBufferSize, "nginx-client-header-buffer-size-in-kb", 0, "Sets buffer size for reading client request header")
```

Comment:
I think we should set the default value equals to ProxyBufferBlocks, because they are both tighly coupled. "16k"